activation/activation.go

```diff
@@ -222,6 +235,53 @@ func (b *Builder) buildNipstChallenge(epoch types.EpochId) error {
 	return nil
 }
 
+func (b *Builder) StartPost(rewardAddress address.Address, dataDir string, space uint64) error {
```

Comment:
You changed `logicalDrive` to `dataDir` and `commitmentSize` to `space` here but not in many other places. It should all be the same, and if the context is lost, you can do `postDatadir` & `postSpace`, or something similar.

Reply:
There are many more leftovers.